desktop/windows/release-notes/index.md

```diff
@@ -17,12 +17,43 @@ redirect_from:
 
 This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
+## Docker Desktop 4.2.0
+2021-11-xx
+
+> Download Docker Desktop
+>
+> [For Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){: .button .primary-btn }
+
+### New
```

Comment:
nit: can you add a blank line after? (mostly for consistency; I think it usually works without, but we had some cases where it caused rendering issues)

```suggestion
### New

```